exercises/beer-song/src/test/scala/BeerSongTest.scala

```diff
@@ -0,0 +1,60 @@
+import org.scalatest._
```

Comment:
Actually I have written a [Code Generator](https://github.com/exercism/xscala/blob/master/testgen/src/main/scala/BeerSongTestGenerator.scala) for this exercise, not being aware at that time that the respective exercise did not exist. :)
 
Don't know if it still works, but feel free to try it out and modify it if you find the time.